### ISSUE TITLE
url-parse version 1.5.0 breaks sockjs-client

### ISSUE THREAD

**Issue by gdelacruzfdez** [sockjs/sockjs-client#532]
Good morning,

Since [url-parse](https://github.com/unshiftio/url-parse) was updated yesterday to version 1.5.0, we are experiencing some issues. Last url-parse version has a bug when extracting the pathname from the url. I have reported the issue already in url-parse repo #https://github.com/unshiftio/url-parse/issues/199

When we instantiate SockJS from http://localhost:3000/PROD/trends:

```javascript
new SockJS(`/dataApi/PROD/ws`);
```
url-parse will use a wrong pathname and the websockets requests are sent to:
```
localhost:3000/PROD/dataApi/PROD/ws
```
instead of:
```
localhost:3000/dataApi/PROD/ws
```
Version 1.4.7 of url-parse works fine.

Not much we can do here, but probably other people are experiencing the same issue. 

On sockjs-client side, the only two solutions are:

- Fix url-parse version to 1.4.7
- Wait until a bugfix is released

**Comment by gdelacruzfdez** [comment]
Fixed in  url-parse 1.5.1

**Comment by brycekahle** [comment]
@gdelacruzfdez Thanks for reporting this. I'm glad `url-parse` got the issue fixed.
